src/api/inbox/inbox-forwarding-user.js

```diff
@@ -0,0 +1,18 @@
+const BASE_PATH = '/unifiedinbox/api/inbox';
+
+export default function(client) {
+  return {
+    updateForwardingConfigurations
+  };
+
+  /**
+   * Update forwarding configurations
+   */
+  function updateForwardingConfigurations(domainId, configurations) {
+    return client.api({
+      url: `${BASE_PATH}/forwardings/configurations?scope=domain&domain_id=${domainId}`,
+      method: 'put',
+      data: { ...configurations }
```

Comment:
Why not simpler like this:

```suggestion
      data: configurations
```